superset/utils/core.py

```diff
@@ -1163,6 +1166,26 @@ def get_username() -> Optional[str]:
         return None
 
 
+def create_temporary_ssl_cert_file(certificate: str) -> str:
```

Comment:
This doesn’t feel temporary to me. Maybe the method and docstring should be updated.

Reply:
Good point. This was a remnant from when I thought these would really be temporary. I will update accordingly.